src/components/TournamentPlanning.vue

```diff
@@ -0,0 +1,26 @@
+<script setup lang="ts">
+import type { TournamentDeref } from '@/models/tournament';
```

Comment:
You missed the `import Content from '@/components/Content.vue';`.

Reply:
Good catch !